.github/workflows/build-native-pr.yml

```diff
@@ -51,3 +55,36 @@ jobs:
           python -m pip install --upgrade pip
           pip install dist/*.whl
           python -c "import pyfory; print(pyfory.__version__)"
+      - name: Upload wheels as artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: pyfory-wheels-native-${{ matrix.os }}-${{ matrix.python-version }}
+          path: dist/*.whl
+
+  publish-to-testpypi:
+    name: Publish to TestPyPI
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download all wheel artifacts
+        uses: actions/download-artifact@v5
+        with:
+          pattern: "pyfory-wheels-native-*"
+          path: downloaded_wheels
+          merge-multiple: true
+
+      - name: Move wheels to a single directory
+        shell: bash
+        run: |
+          mkdir dist
+          find downloaded_wheels -type f -name "*.whl" -exec mv {} dist/ \;
+          ls -R dist
+
+      - name: Publish to TestPyPI
```

Comment:
We can't publish wheel to pypi for every pr. This will generate too much wheels, and pypi doesn't allow overwrite wheel, so most upload will just fail or be skipped.

Reply:
If you want to test it, you can git push a tag to this repo, but this need a committer previlege.

Or you can create a testpypy account, and rename the wheels, then you can upload wheel to that project